Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/HttpObservable.java

```diff
@@ -357,6 +362,7 @@ private SSLConnectionSocketFactory createSSLSocketFactory() {
                 httpRequest.setHeader(entity.getContentType());
             }
 
+            httpRequest.removeHeader(httpRequest.getFirstHeader("Content-Length"));
```

Comment:
>    httpRequest. [](start = 8, length = 16)

Add  note next iteration

Reply:
Do we still need the change for base HttpObservable?